### ISSUE TITLE
Rename px to proxmox

### ISSUE THREAD

**Issue by tabic** [bellackn/ansible-collection-homelab#1]
**Is your feature request related to a problem? Please describe.**
px is not a very descriptive name.

**Describe the solution you'd like**
I would recommend to rename the role to `ansible-role-proxmox-lxc`

**Additional context**
Thank you for this ansible role. This is the only role I found online which supports multiple containers at once and is not bloated with other stuff like downloading templates etc etc. I hope you keep it like that.

**Comment by tabic** [comment]
I suggest some additional renaming to support other roles like some `ansible-role-proxmox-kvm` without the requirement to duplicate variables

px_lxc_api_user -> proxmox_api_user
px_lxc_api_password -> proxmox_api_password
px_lxc_proxmox_host -> proxmox_api_host

**Comment by bellackn** [comment]
Hi @tabic, thanks for the feedback!
I agree that renaming the role would make sense. On my local machine, `px` became an equivalent for Proxmox in many places, but it's really not very descriptive.

About renaming some variables: As there is currently no KVM role planned, I wouldn't like to change them. The way they are, the name reflects clearly to what role a variable belongs and ensures that no namespace issues occur with some other third party role. Also, I'm not sure whether there are cases in which you have dedicated credentials for either managing LXCs or KVMs. In this case, you wouldn't want this namespace collision.

You can still do something like this:
```
proxmox_api_user: foo
proxmox_lxc_api_user: "{{ proxmox_api_user }}"
proxmox_kvm_api_user: "{{ proxmox_api_user }}"
```

**Comment by tabic** [comment]
Good point, makes sense!

**Comment by bellackn** [comment]
This was resolved in #2. I have released a new version 2.0.0 which can be downloaded here: https://galaxy.ansible.com/bellackn/proxmox_lxc
